GitUI/Win32/NativeMethod.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace GitUI.Win32
+{
+    internal static class NativeMethod
```

Comment:
Move to https://github.com/gitextensions/gitextensions/tree/master/GitUI/Interops